.github/workflows/rust.yml

```diff
@@ -362,7 +362,7 @@ jobs:
           key: cargo-cache-
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.11"
```

Comment:
This version matches the version bundled in debian bookworm, on which the rust image is based